packages/expo-clipboard/android/src/main/java/expo/modules/clipboard/ClipboardModule.kt

```diff
@@ -44,6 +99,11 @@ class ClipboardModule : Module() {
 
     onDestroy {
       clipboardEventEmitter.detachListener()
+      try {
+        moduleCoroutineScope.cancel(ModuleDestroyedException())
+      } catch (e: IllegalStateException) {
+        Log.e(TAG, "The coroutine scope has no job in it")
```

Comment:
> Upon every module destruction there would be this this message, because moduleCoroutineScope would have a job only if some method is running at the moment right?

Right. This will throw if no coroutine is running during module destruction. But we need to cancel it anyway to ensure no async methods are executed anymore. But I'm not sure - did you mean the whole `try-cancel-catch` mechanism or just the log message? 😅 
 
I just copied this code from other modules having this functionality (expo-barcode-scanner, expo-image and some others I don't remember).

Reply:
My concern is about logging this error, as this catch would be triggered every time the module is destroyed (and this means every time the app is closed or even more often), so I guess adding this misinformation to the logs (that might be later on analysed, because of some more severe problem) is not right.

Reply:
Removed the log, just left a comment instead